documentation/faq.asciidoc

```diff
@@ -279,3 +279,13 @@ The solution is to indirectly enable the `log_bin` property, activating certain
 
 == Why Debezium PostgreSQL connector causes abnormal consumption of WAL database disk space?
 See link:/documentation/reference/1.0/connectors/postgresql.html#wal-disk-space[WAL Disk Space Consumption] in PostgreSQL connector documentation.
+
+== Why a new schema version is not created when the content of two columns in a table is swapped?
+If two columns in a table are swapped in that way that after the change the table schema is same as before then no new version of schema is not created in schema registry.
```

Comment:
```suggestion
If two columns in a table are swapped in that way that after the change the table schema is same as before, then no new version of the schema gets created in the schema registry.
```

Does this actually only apply to the schema registry, or also to the schema embedded in JSON events?

Reply:
@gunnarmorling JSON schemas are not versioned, they are not kept anywhere so there is no way how to version them